erpnext/hr/doctype/expense_claim/expense_claim.py

```diff
@@ -34,3 +52,19 @@ def validate_expense_approver(self):
 		if self.exp_approver and "Expense Approver" not in frappe.get_roles(self.exp_approver):
 			frappe.throw(_("{0} ({1}) must have role 'Expense Approver'")\
 				.format(get_fullname(self.exp_approver), self.exp_approver), InvalidExpenseApproverError)
+	
+	def update_task(self):
```

Comment:
Write this function in task.py and use `frappe.get_doc("Task", self.task).update_total_expense_claim()`